server/controllers/ParcelController.js

```diff
@@ -0,0 +1,51 @@
+import parcelModel from '../models/ParcelModel';
+
+class parcels {
+  // get all parcel items
+  static async getItems (req, res) {
```

Comment:
`async` and `await` do the same thing with `.then` and `.catch`, both are returning a promise. Is better you should use one. 
eg -> this piece of code those the same thing